src/elements/molecules/Carousel/Carousel.css

```diff
@@ -0,0 +1,72 @@
+:root {
+
+  /* CSS variables */
+}
+
+.carousel {
+
+  /* Baseline styles */
+  margin: rem(16px) auto;
+
+  &__controls {
+    justify-content: center;
+
+    .carousel__button {
+      margin: auto rem(8px);
+    }
+  }
+
+  &__pagination:not(.carousel__pagination-custom),
+  .tns-nav {
+    text-align: center;
+
+    button {
+      background-color: color(var(--color-gray-light), 0.5);
+      border: 1px solid;
+      border-color: var(--color-gray);
+      border-radius: 50%;
+      height: rem(16px);
+      margin: rem(5px);
+      width: rem(16px);
+    }
+  }
+
+  .tns-nav-active {
+    background-color: var(--color-gray) !important;
```

Comment:
If we move .tns-nav-active to be nested within the .tns-nav button on line 23, and prefix it as**&.tns-nav-active** we wont need to use an !important. 

Normally, the way you have it now is correct; in that you want to avoid nesting that far, however this is ok here to overcome the strength of the .tns-nav button selector's background-color setting.